lib/uvobot/uvo_parser.rb

```diff
@@ -27,13 +27,27 @@ def self.parse_detail_link(tr_node, bulletin_url)
 
     def self.parse_detail(html)
       # there are multiple formats of detail page, this method does not handle them all for now
-      h_doc = doc(html)
-      amount_nodes = h_doc.xpath('//div[text()="Hodnota            "]')
+      result = {
+        amount: parse_amount(html),
+        procurement_type: parse_procurement_type(html)
+      }
+
+      result.values.uniq == [nil] ? nil : result
```

Comment:
Dohodnute spravanie parse_detail bolo, ze ak nenájde info o detaile tak vrati nil. Takze ked sa nepodari ani jednu detailnu info najst tak vratim nil ako predtym.

Ci teraz uz by to mohol byť aj hash s nil hodnotami po novom?